pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/rebalance/RebalanceConfig.java

```diff
@@ -32,9 +32,9 @@ public class RebalanceConfig {
   public static final long DEFAULT_EXTERNAL_VIEW_CHECK_INTERVAL_IN_MS = 1000L; // 1 second
   public static final long DEFAULT_EXTERNAL_VIEW_STABILIZATION_TIMEOUT_IN_MS = 3600000L; // 1 hour
 
-  // Whether to rebalance table in dry-run mode
+  // Whether to rebalance table in dry-run mode.
   @JsonProperty("dryRun")
-  @ApiModelProperty(example = "false")
+  @ApiModelProperty(example = "true")
```

Comment:
why are these changes needed?

Reply:
I could be wrong here, but isn't this just the example shown? or it actually uses the example as the default?